src/main/java/com/cburch/logisim/gui/log/LogFrame.java

```diff
@@ -148,13 +149,13 @@ private static String computeTitle(Model data, Project proj) {
 
   private static final long serialVersionUID = 1L;
   private Simulator curSimulator = null;
-  private Model curModel;
+  @Getter private Model model;
```

Comment:
The dropped `cur` prefix seems to be on purpose to relate the member variable to `curSimulator`. Also, type and variable names differing only in casing might provoke coding errors.